specification/_types/mapping/core.ts

```diff
@@ -241,11 +242,33 @@ export class SearchAsYouTypeProperty extends CorePropertyBase {
   type: 'search_as_you_type'
 }
 
+/**
+ * A variant of text that trades scoring and efficiency of positional queries for space efficiency. This field
+ * effectively stores data the same way as a text field that only indexes documents (index_options: docs) and
+ * disables norms (norms: false). Term queries perform as fast if not faster as on text fields, however queries
+ * that need positions such as the match_phrase query perform slower as they need to look at the _source document
+ * to verify whether a phrase matches. All queries return constant scores that are equal to 1.0.
+ */
+export class MatchOnlyTextProperty {
+  type: 'match_only_text'
+  /**
+   * Multi-fields allow the same string value to be indexed in multiple ways for different purposes, such as one
+   * field for search and a multi-field for sorting and aggregations, or the same string value analyzed by different analyzers.
+   * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html
+   */
+  fields?: Dictionary<PropertyName, Property>
+  /**
+   * Metadata about the field.
+   * @doc_id mapping-meta-field
+   */
+  meta?: Dictionary<string, string>
+}
```

Comment:
`copy_to` is also applicable to this field mapping:

```suggestion
  /**
   * Allows you to copy the values of multiple fields into a group
   * field, which can then be queried as a single field.
   */
  copy_to?: Fields
}
```